dev-util/smartgit/smartgit-6.5.6.ebuild

```diff
@@ -7,23 +7,27 @@ EAPI=5
 inherit eutils java-utils-2 versionator
 
 MY_PV="$(replace_all_version_separators _)"
-MY_P="${PN}hg-generic-${MY_PV}"
+MY_P="${PN}-generic-${MY_PV}"
 
 DESCRIPTION="Git client with support for GitHub Pull Requests+Comments, SVN and Mercurial"
 HOMEPAGE="http://www.syntevo.com/smartgit"
-SRC_URI="http://www.syntevo.com/download/${PN}hg/${MY_P}.tar.gz"
+SRC_URI="http://www.syntevo.com/download/${PN}/${MY_P}.tar.gz"
 
 SLOT="0"
 LICENSE="smartgit"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
+IUSE="git mercurial"
```

Comment:
This is not the correct way. Pure optional runtime packages should not be added as USE based dependency but you should either hard depend on it or use optfeature() from eutils.eclass